tests/conftest.py

```diff
@@ -260,3 +261,42 @@ def gs_sim(
     scene.build()
 
     return gs_sim
+
+
+@pytest.fixture
+def box_obj_path(tmp_path):
```

Comment:
I think you could reuse this fixture in tests that were merged in some of your previous PRs no? To avoid code duplication.

Reply:
I see… do as you think is best!